specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+    "version": "3.0"
+  },
+  "host": "westus.api.cognitive.microsoft.com",
+  "basePath": "/speaker/identification/v3.0",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    },
+    "apiKeyQuery": {
+      "type": "apiKey",
+      "name": "subscription-key",
+      "in": "query"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    },
+    {
+      "apiKeyQuery": []
+    }
+  ],
+  "x-servers": [
+    {
+      "url": "https://westus.api.cognitive.microsoft.com"
+    },
+    {
+      "url": "https://cognitive-wu.cognitiveservices.azure.com"
+    },
+    {
+      "url": "https://api.ProjectOxford.ai"
+    }
+  ],
+  "parameters": {
+    "profileIdParam": {
+      "in": "path",
+      "name": "profileId",
+      "required": true,
+      "description": "Unique identifier for profile id (guid).",
+      "type": "string",
+      "pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$"
+    },
+    "localeParam": {
+      "in": "path",
+      "name": "locale",
+      "required": true,
+      "type": "string",
+      "pattern": "^[a-zA-Z]{2}-[a-zA-Z]{2}$",
+      "description": "A combination of language code and country code."
+    }
+  },
+  "definitions": {
+    "ProfileId": {
+      "type": "string",
+      "pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$",
+      "description": "Unique identifier for profile id (guid).",
+      "example": "49a36324-fc4b-4387-aa06-090cfbf0064f"
+    },
+    "Locale": {
+      "type": "string",
+      "pattern": "^[a-zA-Z]{2}-[a-zA-Z]{2}$",
+      "description": "Language identifier consisting of a combination of language code and country code.",
+      "example": "en-US"
+    },
+    "ProfileStatus": {
+      "type": "string",
+      "description": "Status representing the current state of the profile. Available values are:\n* Active: profile is active and can be used if the enrollment status is 'Enrolled'.\n* Inactive: profile has not been activated and an activation phrase must be submitted.",
+      "enum": [
+        "Active",
+        "Inactive"
+      ],
+      "example": "Active"
+    },
+    "EnrollmentStatus": {
+      "type": "string",
+      "description": "Status representing the current state of the enrollment. Available values are:\n* Enrolling: profile has no voice print and not ready for recognition requests.\n* Training: voice print of profile is being created and can’t be used for recognition at the moment.\n* Enrolled: profile has a voice print and ready for recognition requests.",
+      "enum": [
+        "Enrolling",
+        "Training",
+        "Enrolled"
+      ],
+      "example": "Enrolling"
+    },
+    "CreatedDateTime": {
+      "type": "string",
```

Comment:
Date-time values like this should have `format: date-time`.

Reply:
done